riscv-sbi.adoc

```diff
@@ -330,9 +330,16 @@ value for this CSR.
 
 == Legacy Extensions (EIDs #0x00 - #0x0F)
 
-The legacy SBI extensions ignores the SBI function ID field, instead being
-encoded as multiple SBI extension IDs. Each of these extension IDs must be
-probed for directly.
+The legacy SBI extensions:
+
+* Ignores the SBI function ID field, since these are encoded as multiple SBI
+  extension IDs
+* Returns 0 in `a0` register upon success
```

Comment:
I think you need to move the return value into the definition of each function, rather than having it up here, because not all of them return 0.  console_getchar returns the unsigned character value, or -1 if no character is present.  clear_ipi returns zero if no IPI had been pending, or an implementation-defined positive value if an IPI had been pending.

Reply:
Sure, I will update like you suggested.